test/logflare_grpc/trace/transform_test.exs

```diff
@@ -0,0 +1,61 @@
+defmodule LogflareGrpc.Trace.TransformTest do
+  use Logflare.DataCase
+  alias LogflareGrpc.Trace.Transform
+
+  describe "to_log_events/1" do
+    setup do
+      user = build(:user)
+      source = insert(:source, user: user)
+      %{resource_spans: TestUtils.random_resource_span(), source: source}
+    end
+
+    test "Creates LogEvent from a list with one Resource Span that contains an Event", %{
+      resource_spans: resource_spans,
+      source: source
+    } do
```

Comment:
here I'm testing that the received struct is properly converted into the correct amount and type of LogEvent (one span can have multiple events for example)

as such it's important to test if each type of event has the expected field values otherwise it will break our next steps of the implementation